packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -483,6 +485,17 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
 
         drawStaticOval(canvasEl!, videoEl!, videoMediaStream!);
       },
+      hideOvalCanvas: (context) => {
+        const { canvasEl } = context.videoAssociatedParams!;
+
+        clearOvalCanvas({ canvas: canvasEl! });
+        canvasEl!.style.display = 'none';
+      },
+      showOvalCanvas: (context) => {
+        const { canvasEl } = context.videoAssociatedParams!;
+
+        canvasEl!.style.display = 'unset';
+      },
```

Comment:
Why does this need to be handled through the state machine? Can we scope this logic to the UI layer?

Reply:
Actually realized we don't need this in the machine layer anyways, removed as suggested